snuba/cli/migrations.py

```diff
@@ -30,7 +33,7 @@ def list() -> None:
     Lists migrations and their statuses
     """
     setup_logging()
-    check_clickhouse_connections()
+    check_clickhouse_connections(CLUSTERS)
```

Comment:
Doesn't this also have to filter CLUSTERS based on readiness state?

Reply:
This command does not take readiness states as an rgument and is only really used locally which is why I didn't bother.